### ISSUE TITLE
feature：Increases the maximum screen size of the window

### ISSUE THREAD

**Issue by hesj** [eczarny/spectacle#940]
Increases the maximum screen size of the window

I need a way to spread the window all over the monitor, but not the full screen. That means the current window size is the largest.

**Comment by batthias** [comment]
I would need this "fake full screen" feature too.

Switched to Rectangle for now. Is this app developed anymore? Rectangle does the same thing and has more options, right?

**Comment by xdvrx1** [comment]
In my understanding, you want to full screen but not the full screen where the windows will be occupying all of the screen and temporarily hiding the commands on top and at the bottom. If that is the case, if that is one Window to be spread, simply double click the window and it will do the job. But, if you are referring to say 3 windows that will take all the possible full screen in just one click, I think, that is not a feature of Spectacle.

**Comment by batthias** [comment]
@xdvrx1 Yes exactly. I should have been more precise about what I mean with "make full screen". Yes the command should just do what happens when you double click on the window.

The option of resizing all windows on the screen so they cover the screen without overlap would be cool too, but does sound harder to implement and is probably not necessary often enough to warrant the development time.

**Comment by emahuni** [comment]
What I was thinking you meant was filling up all available space... more like one Window taking up what top-half and bottom-half would take. No fancy hiding of anything. I thought this was supposed to be there when I was looking all over and failed to find it. I think it's a pretty straightforward implementation. This tool is just amazing, thank you sooooo much.

**Comment by emahuni** [comment]
PS. I just noticed that what you are calling fullscreen is actually what I wanted. The naming is wrong, it should be maximize or zoom or something along those lines.
